### PR TITLE
alembic: addition of dependency to invenio-records

### PR DESCRIPTION
**Pull request by dinosk**
(closes [#41](https://github.com/inveniosoftware/invenio-records-files/issues/41))

Signed-off-by: Dinos Kousidis <konstantinos.kousidis@cern.ch>

### CODE REVIEW

**Comment by nharraud** [comment]
Thanks @dinosk  👍 
Looks like the "lowest" build fails on travis. Could you try to fix it in an additional commit? As the "release" build passes, updating the invenio dependencies to their last version in the `setup.py` is probably enough.

**Comment by dinosk** [comment]
Yes true, I changed the first dependency from invenio-db to invenio-files-rest, as invenio-files-rest is itself dependent on invenio-db and I thought it wasn't needed here. The second commit upgrades the version of Sphinx as the break in "lowest" was due to it.

File: invenio_records_files/alembic/2da9a03b0833_create_records_files_branch.py [hunk 1]
```diff
@@ -31,7 +31,7 @@
 revision = '2da9a03b0833'
 down_revision = 'dbdbc1b19cf2'
 branch_labels = (u'invenio_records_files',)
-depends_on = 'dbdbc1b19cf2'
+depends_on = ('52ce868f33c3', '862037093962')
```
Comment (nharraud) [inline invenio_records_files/alembic/2da9a03b0833_create_records_files_branch.py]: We need to create the dependency between [1ba76da94103](https://github.com/inveniosoftware/invenio-records-files/blob/9f612d043de6d19ac4e8b881909c4793ff81deb4/invenio_records_files/alembic/1ba76da94103_create_records_files_tables.py) and ([2e97565eba72](https://github.com/inveniosoftware/invenio-files-rest/blob/f4230691ab8ca0479ee672a5fa73795f582e9a1b/invenio_files_rest/alembic/2e97565eba72_create_files_rest_tables.py), [862037093962](https://github.com/inveniosoftware/invenio-records/blob/bb085fb417409296de2291f65a2de3c4f9702ae3/invenio_records/alembic/862037093962_create_records_tables.py))

Comment (nharraud) [inline invenio_records_files/alembic/2da9a03b0833_create_records_files_branch.py]: Also if you can format it like this:
```
depends_on = (
  '...', # invenio-files-rest
  '...', # invenio-records
)
```

File: tests/test_invenio_records_files.py [hunk 2]
```diff
@@ -308,3 +308,22 @@ def test_index_attachments(app, db, location, record_with_bucket):
...
+        raise pytest.skip('Upgrades are not supported on SQLite.')
+
+    assert not ext.alembic.compare_metadata()
+    db.drop_all()
+    ext.alembic.upgrade()
+
+    assert not ext.alembic.compare_metadata()
+    ext.alembic.stamp()
+    ext.alembic.downgrade(target='96e796392533')
+    ext.alembic.upgrade()
```
Comment (dinosk) [inline tests/test_invenio_records_files.py]: Indeed commenting this out makes the test fail. [compare_metadata()](https://bitbucket.org/davidism/flask-alembic/src/e036f063af1ba667c74560264639a93d812dfd51/flask_alembic/extension.py?at=default&fileviewer=file-view-default#extension.py-469) returns the diff list that [produce_migrations()](https://bitbucket.org/davidism/flask-alembic/src/e036f063af1ba667c74560264639a93d812dfd51/flask_alembic/extension.py?at=default&fileviewer=file-view-default#extension.py-463) produces.

**Pull request merged**